pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "numpy==1.23.5",
     "pandas==1.5.1",
     "transformers==4.25.1",
-    "promptsource==0.2.3",
+    "git+https://github.com/bigscience-workshop/promptsource.git",
```

Comment:
Any reason, of why we want always the latest version of promptsource? Making sure that we have all the datasets? (Just asking, because this could reguallry maybe break things?)
(I have also a [_custom_ promptsource](https://github.com/NotodAI-Research/promptsource) to use "unofficial" datasets, which we might want to include into our project as submodule maybe.)

Reply:
0.2.3 only work with python 3.7

Reply:
but maybe your promptsource doens't? I can switch to yours if it's better

Reply:
I think we definitely shouldn't list `promptsource==0.2.3` because it literally doesn't work on the Python versions that we recommend/require (3.9 and above). I'm fine with either `git+https://github.com/bigscience-workshop/promptsource.git` or `git+https://github.com/NotodAI-Research/promptsource.git`.

Reply:
Let's link to git+https://github.com/bigscience-workshop/promptsource.git for now. I'll clean up my promptsource a bit and linkt to that later